test/people-route-test.js

```diff
@@ -0,0 +1,204 @@
+          expect(res.status).to.equal(400);
+          done();
+        });
+      });
+    });
+  });
+
+  describe('GET: /api/people/:id', function() {
+    describe('with a valid body', function() {
+      before( done => {
+        new People(examplePeople).save()
+        .then( people => {
+          this.tempPeople = people;
+          return People.findByIdAndAddPresident(people._id, examplePresident)
+        })
+        .then( president => {
+          this.tempPresident = president;
+          done();
+        })
+        .catch(done);
+      });
+
+      after( done => {
+        if(this.tempPeople) {
+          People.remove({})
+          .then(() => done())
+          .catch(done);
+          return;
+        };
+        done();
+      });
+
+      it('should return an actual people', done => {
+        request.get(`${url}/api/people/${this.tempPeople._id}`)
+        .end((err, res) => {
+          if(err) return done(err);
+          expect(res.status).to.equal(200);
+          expect(res.body.name).to.equal('test people name');
+          expect(res.body.presidents.length).to.equal(1);
+          done();
+        });
+      });
+    });
+
+    describe('with an invalid id', function() {
+      it('should return a 404 error', function() {
+        request.get(`${url}/api/people/121212`)
+        .end((err, res) => {
+          expect(res.status).to.equal(404);
+          done();
+        });
+      });
+    });
+  });
+
+  describe('PUT: api/people/:id', function() {
+    describe('with a valid id and body', function() {
+      before( done => {
+        new People(examplePeople).save()
+        .then( people => {
+          this.tempPeople = people;
+          done();
+        })
+        .catch(done);
+      });
+
+      after( done => {
+        if(this.tempPeople) {
+          People.remove({})
+          .then(() => done())
+          .catch(done);
+          return;
+        };
+        done();
+      });
+
+      it('should update a people with a valid id and body', done => {
+        let updatePeople = {name: 'some new people'}
+        request.put(`${url}/api/people/${this.tempPeople._id}`)
+        .send(updatePeople)
+        .end((err, res) => {
+          if(err) return done(err);
+          let timestamp = new Date(res.body.timestamp);
+          expect(res.status).to.equal(200);
+          expect(timestamp.toString()).to.equal(examplePeople.timestamp.toString());
+          expect(res.body.name).to.equal(updatePeople.name);
+          done();
+        });
+      });
+    });
+
+    describe('with an invalid id', function() {
+      it('should return a 404 error with an invalid id', function() {
+        request.put(`${url}/api/people/28288282`)
+        .end((err, res) => {
+          expect(res.status).to.equal(404);
+          done();
+        });
+      });
+    });
+
+    describe('with no body supplied', function() {
+      it('should return a 400 error', function() {
+        request.put(`${url}/api/people`)
```

Comment:
You need to pass a ID that you want to update otherwise the 400 error will be caught first by a 404 error because you didn't provide an ID param.

Additionally even after passing a valid ID to update you need to write a conditional that checks if the req.body has any content. An update that includes an empty body is technically not prohibited unless you write a conditional that catches a 400 error.